crypto/include/internal/ec_int.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2018 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
@mattcaswell Is the copyright line ok? Or should it be
~~~
 * Copyright 2018-2018 The OpenSSL Project Authors. All Rights Reserved.
~~~

Reply:
Just one year is fine.  Our script updates the copyrights on all files touched anyway so don't worry too much.

Reply:
Thanks @richsalz